### PR TITLE
Use EmailRequestModel in AssessmentNotificationsDispatch

### LINKED ISSUE #161: AssessmentNotificationsDispatch should use the EmailRequestModel
Currently the AssessmentNotificationsDispatch uses the AssessmentNotification when sending requests to the Email Service. This needs to be changed to the EmailRequestModel.

### PR DESCRIPTION
**Pull request by irinel-nistor**
### What does it fix?

Closes #161

I changed the AssessmentNotificationsDispatch.Process message to send a EmailModelRequest

### How has it been tested?

I did a manual test. I changed the scheduler to run every minute and debugged to see that the EmailModelRequet clas gets created and send

### CODE REVIEW

**Review by RaduCStefanescu — APPROVED** [review]

**Comment by RaduCStefanescu** [comment]
@trupci can you give some input on this PR?
The email of an user should be taken from the entity owned by the Identity Server, not from the `UserInfo` in the API.

File: backend/src/StamAcasa.Common/DTO/UserInfo.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@ public class UserInfo {
         [JsonIgnore]
         public int? ParentId { get; set; }
         public string Name { get; set; }
+        public string Email { get; set; }
```
Comment (RaduCStefanescu) [inline backend/src/StamAcasa.Common/DTO/UserInfo.cs:11]: This information should not be displayed in API, we should add `JsonIgnore` to this field.

**Review by RaduCStefanescu — CHANGES_REQUESTED** [review]
Please check out the comment added.

File: backend/src/StamAcasa.Common/DTO/UserInfo.cs (hunk 1 above)
Comment (irinel-nistor) [reply backend/src/StamAcasa.Common/DTO/UserInfo.cs:11]: Thanks, nice catch! Sorry about it, I pushed a fix.

**Review by RaduCStefanescu — APPROVED** [review]

**Pull request merged**